coding/python_style_guide.rst

```diff
@@ -662,25 +686,39 @@ This means ``/`` is floating-point division and ``//`` is truncated integer divi
 This gives more predictable behavior than the old operators, avoiding a common source of obscure bugs.
 It also makes intent of the code more obvious.
 
-.. _style-guide-py-10-2:
+.. _style-guide-py-future-absolute-import:
 
 Use ``from __future__ import absolute_import``
 ----------------------------------------------
 
 In addition, import local modules using relative imports (e.g. ``from . import foo`` or ``from .foo import bar``).
 This results in clearer code and avoids shadowing global modules with local modules.
-It also makes 2to3_ conversion more reliable.
 
-.. _style-guide-py-10-3:
+.. _style-guide-py-exception-as:
+
+Use ``as`` when catching an exception
+-------------------------------------
+
+For example, use ``except Exception as e`` or ``except (LookupError, TypeError) as e``.
+The new syntax is clearer, especially when catching multiple exception classes, and required in Python 3.
+
+.. _style-guide-py-generators:
+
+Iterators and generators SHOULD be used to iterate overlarge data sets efficiently
+----------------------------------------------------------------------------------
+
+Use iterators, generators (classes that act like iterators) and generator expressions (expressions that act like iterators) to iterate over large data sets efficiently.
+
+.. _style-guide-py-future-itervalues:
 
 Use ``itervalues()`` and ``iteritems()`` instead of ``values()`` and ``items()``
 --------------------------------------------------------------------------------
 
 For iterating over dictionary values and items use the above idiom unless you truly need a list.
-This generates more efficient code today and helps 2to3_ generate more efficient code in the future.
-For more information see http://python3porting.com/preparing.html#optional-use-the-iterator-methods-on-dictionaries.
+This generates more efficient code today and helps futurize_ generate more efficient code in the future.
+For more information see http://python-future.org/compatible_idioms.html#iterating-through-dict-keys-values-items.
```

Comment:
with the caveat "unless the code has already been ported to support Python 3". `meas_algorithms` and below can not use `itervalues` etc.